### ISSUE TITLE
Speedometer example?

### ISSUE THREAD

**Issue by beyonlo** [quitegreensky/akivymd#5]
Hello, nice work!

Please, could you to create a simple Speedometer example to start?

Thank you.

**Comment by quitegreensky** [comment]
Hey! thanks for suggestion. sounds like a good idea. could you share a picture of that?

**Comment by beyonlo** [comment]
Hi  @quitegreensky 

> Hey! thanks for suggestion. sounds like a good idea. could you share a picture of that?

I'm using this kivy speedmeter ( https://github.com/kivy-garden/speedmeter ). But it is very ugly and has some bugs, as I reported in that project https://github.com/kivy-garden/speedmeter/issues/1

I would like a example very beautiful speedometer. I do not found a beautiful picture for you now, but tried to find on internet some examples (are attached), but somes is not so beautiful too, but as reference is good.
Here has other options too: https://www.vectorstock.com/royalty-free-vectors/speedometers-vectors

I remembered about car speed Android App, developed using kivy, where has some beautiful speedometer, but I not remember the app name. Anyway, I think this is fine for you, for now :)

**Important:**
1. Anyway, I suggest you to do a speedmeter where is possible to configure it to be 360, 180 or 90 degrees, because when used in small devices, the 180 and 90 degrees are very good options to optimize screen space.
2. to be possible to configure tick and subtick, like as is possible to do with that https://github.com/kivy-garden/speedmeter


I will to try still to found a beautiful speedometer to send you.

Thank you so much :)


![power-gauge-meter-stages-eps-vector_csp19653387](https://user-images.githubusercontent.com/630637/91342330-3f924680-e7b1-11ea-93f9-38fa6d61251e.jpg)
![black-speedometer-scale-speed-gauge-vector-16749487](https://user-images.githubusercontent.com/630637/91342332-402add00-e7b1-11ea-91d0-610db6209621.jpg)
![futuristic-car-speedometers-speed-eps-vector_csp55818647](https://user-images.githubusercontent.com/630637/91342334-402add00-e7b1-11ea-9934-0187706fa57a.jpg)
![futuristic-car-speedometer-speed-hud-kilometer-vector-22579459](https://user-images.githubusercontent.com/630637/91342336-40c37380-e7b1-11ea-8054-b784909e7eba.jpg)

**Comment by beyonlo** [comment]
Hi @quitegreensky 

Do you have a plan to put the speedometers on the roadmap? :)

Thanks!

**Comment by quitegreensky** [comment]
@beyonlo Yes and that's why this issue is labeled as enhancement. Creating a simple speedometer is fairly easy but the ones that you expect will take time.

**Comment by beyonlo** [comment]
> @beyonlo Yes and that's why this issue is labeled as enhancement. Creating a simple speedometer is fairly easy but the ones that you expect will take time.

@quitegreensky 

No problem, I will wait for :)

As a suggestion, create two kinds of speedometers:
1. Simple speedometer.
2. More complex speedometer.

Obs: if possible, in the both kind of speedometers to be possible to configure to be 360, 180 or 90 degrees

Thank you so much.
